packages/bot/src/scratch/xml/toolbox.xml

```diff
@@ -7,6 +7,7 @@
         description="Trade duration, barrier, stake"
         iconURI="ic-stage-1"
     >
+        <button text="A button" callbackKey="myFirstButtonPressed"></button>
```

Comment:
is this for testing purpose? If so, we can remove?

Reply:
Yeah we can remove this. 👍